src/Controls/src/Core/Compatibility/Handlers/Android/FrameRenderer.cs

```diff
@@ -14,6 +14,7 @@
 
 namespace Microsoft.Maui.Controls.Handlers.Compatibility
 {
+	[Obsolete("Frame is obsolete as of .NET 9. Please use Border instead.")]
```

Comment:
Would be a good idea to hide it from users with `[EditorBrowsable(EditorBrowsableState.Never)]`?

Reply:
Not against it, but I also don't think it adds a ton of value. The obsolete should show up as a warning and that should be enough. Maybe if anything people will even be confused if they suddenly can't find Frame anymore.

Reply:
Yea, and we're generally just trying to get away from EditorBrowsable